broadcast/dtos/dtos.go

```diff
@@ -0,0 +1,75 @@
+package dtos
+
+import (
+	"context"
+	"google.golang.org/protobuf/reflect/protoreflect"
+	"time"
+)
+
+type Msg interface {
+	GetBroadcastID() uint64
+	GetMethod() string
+	String() string
+}
+
+type BroadcastMsg struct {
+	Ctx        context.Context
```

Comment:
Difficult to assess, since I don't know the use of the `Ctx` field here, but it is a antipattern to include `context.Context` in structs. See the [Context and Structs blog post](https://go.dev/blog/context-and-structs). It should be passed as the first argument to all functions/methods that need a context. But see the blog for concrete advice.